src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -3213,14 +3213,16 @@ private BoundExpression BindArgumentExpression(BindingDiagnosticBag diagnostics,
         }
 
 #nullable enable
-        private void CoerceArguments<TMember>(
+        private void CheckAndCoerceArguments<TMember>(
             MemberResolutionResult<TMember> methodResult,
-            ArrayBuilder<BoundExpression> arguments,
+            AnalyzedArguments analyzedArguments,
             BindingDiagnosticBag diagnostics,
-            BoundExpression? receiver)
+            BoundExpression? receiver,
+            bool interpolatedStringHandler = false)
```

Comment:
>interpolatedStringHandler [](http://example.com/codeflow?start=17&length=25)

There is an alternative to adding an extra parameter. At the affected call sites, we could use a dedicated ```BindingDiagnosticBag``` and filter out warnings by id in the process of merging diagnostics into the original bag. Consider if you prefer doing that instead.

Reply:
Yeah, sounds better than passing a parameter and worrying if we passed it to all callers.